requirements.dev.txt

```diff
@@ -4,3 +4,7 @@ sphinx-autodoc-typehints
 recommonmark
 pypandoc
 twine
+
+pytest
```

Comment:
these test requirements should go to tests_require in setup.py (and are actually already there). The requirements here are additional requirements on top of that.